File: level2/최댓값과 최솟값/Solution.java
class Solution {
    public String solution(String s) {
        String[] arr = s.split(" ");
        for (int i = 0; i < arr.length; i++) {
            for (int j = i + 1; j < arr.length; j++) {
                if (Integer.parseInt(arr[i]) > Integer.parseInt(arr[j])) {
                    String tmp = arr[j];
                    arr[j] = arr[i];
                    arr[i] = tmp;
                }
            }
        }
        return arr[0] + " " + arr[arr.length - 1];
    }
}
 

File: level2/최댓값과 최솟값/Solution.py
def solution(s):
    list = s.split(' ')
    intList = [int(list[i]) for i in range(len(list))]
    return str(min(intList))+' '+str(max(intList))
  
  '''
  def solution(s):
    li = list(map(int, s.split()))
    return f'{min(li)} {max(li)}'
  '''
